Exercise_2.js

```diff
@@ -0,0 +1,272 @@
+const inventors = [
+  "Albert Einstein",
+  "Issac Newton",
+  "Galileo Galilei",
+  "Marie Curie",
+  "Johannes Kepler",
+  "Nicolaus Copernicus",
+  "Max Planck",
+  "Katherine Blodgett",
+  "Ada Lovelace",
+  "Sarah E. Goode",
+  "Lise Meitner",
+  "Hanna Hammarstrom"
+];
+
+const numbers = [1, 2, 3, 5, 8, 13, 21, 34, 55];
+////////////////////////some/every/////////////////////
+// Does any inventor have the letter 'y' in their name?
+result = inventors.some(function (name) {
+  return name.includes('y')
+});
+console.log(result);
+
+
+// Does every inventor have the letter 'e' in their name?
+result = inventors.every(function (name) {
+  return name.includes('e')
+});
+console.log(result);
+
+
+// Does every inventor have first name that's longer than 4 characters?
+result = inventors.some(function (name) {
+  var fname = name.split(" ")[0]
+  return fname > 4
+});
+console.log(result)
+
+/////////////////filter////////////////////////////////////
+// Print out an array of the inventors whose name starts with 'A'.
+result = inventors.filter(function (name) {
+  return name[0] === ("A")
+})
+console.log(result);
+
+
+// Print out an array of the inventors whose name contains 'n'.
+result = inventors.filter(function (name) {
+  return name.includes("n")
+});
+console.log(result);
+
+
+// Print out an array of the inventors whose name has the same letter twice in a row (e.g. nn or mm).
+
+result = inventors.filter(function (name) {
+  for (var i = 0; i < name.length; i++) {
+    if (name[i] === name[i + 1]) {
+      return name
+    }
+  }
+});
+console.log(result);
+
+// Print out an array of the numbers which are odd.
+result = numbers.filter(function (number) {
+  if ((number % 2 === 0) && (number > 0)) {
+    return number
+  }
+});
+console.log(result);
+
+
+
+// Print out an array of the numbers that have two digits.
+result = numbers.filter(function (number) {
+  if ((number > 9) && (number < 100)) {
+    return number
+  }
+});
+console.log(result);
+
+
+// Print out an array of the numbers which are prime.
+result = numbers.filter(function (number) {
+  if (number <= 1) {
+    return false
+  }
+  for (var i = 2; i <= (Math.sqrt(number)); i++) {
+    if (number % i === 0) {
+      return false;
+    }
+    return true;
+  }
+});
+console.log(result)
```

Comment:
What, are you sure about that, please check the result of exercise before push code. 
21 and 55 is not the prime numbers.
<img width="271" alt="image" src="https://user-images.githubusercontent.com/79957186/197356621-d444ad5a-10d8-4fde-b774-e7b74f7d80d9.png">